clouds/databricks/Makefile

```diff
@@ -88,3 +97,13 @@ ci-release-core:
 
 create-headers:
 	$(MAKE) -C libraries/scala create-headers
+
+create-package: clean build
+	echo "- Creating package"
+	rm -rf $(DIST_DIR)
+	mkdir -p $(DIST_DIR)/$(PACKAGE_NAME)
+	cp $(BUILD_DIR)/modules.sql $(DIST_DIR)/$(PACKAGE_NAME)/
+	echo '{"latest_version": "$(PACKAGE_VERSION)"}' > $(DIST_DIR)/metadata.json
+	cd $(DIST_DIR) && zip -r $(PACKAGE_NAME).zip . && cd ..
+	cp $(BUILD_DIR)/modules.sql $(DIST_DIR)/modules-$(PACKAGE_VERSION).sql
```

Comment:
Why is this required `modules-$(PACKAGE_VERSION).sql`?

Reply:
removed now